CHANGELOG.rst

```diff
@@ -7,6 +7,17 @@
 Changelog
 =========
 
+1.1.0 - 2022.06.01
+------------------
+
+**New feature: Statistical Tests**
+
+- Implemented a :class:`~datajudge.BetweenRequirement` constraint running a Kolmogorov Smirnov test.
+
+**New method: get_column**
```

Comment:
I'd omit this one. It's an internal detail of the library and not part of what a user of this library needs in order to make data tests.

Reply:
Oh okay, I thought the changelog is also directed towards possible developers wanting to extend the library -- either officially or with their own constraints.

Reply:
I thought the titles `**title**` are splitting sections and that made it clear. I'm new to `.rst` files, is there any convention we're following or I can lookup?
At first glance, I found [this](https://www.ros.org/reps/rep-0132.html), which is, of course, from a different domain but was nevertheless referenced in a few places.

Reply:
Regarding conversation resolving: Sorry! I was cleaning up because I was losing track of some of the tasks. Makes sense of course; basic PR etiquette.

Reply:
No convention. And indeed you were having sections, but I meant not to just say "new features" or "new method" but to split into a user relevant section and a developer relevant section.